envoy/service/discovery/v2/BUILD

```diff
@@ -2,91 +2,20 @@ load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library"
 
 licenses(["notice"])  # Apache 2
```

Comment:
We should probably leave a note explaining why only ADS/HDS/SDS live here for posterity.